docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/druva-source.md

```diff
@@ -0,0 +1,151 @@
+
+On the [Collection page](/docs/manage/health-events#collection-page), the Health and Status for Sources is displayed. Use [Health Events](/docs/manage/health-events.md) to investigate issues with collection. You can click the text in the Health column, such as **Error**, to open the issue in Health Events to investigate.
+
+
+## Setup and configuration Source
+
+When you create a Druva Source, you add it to a Hosted Collector. Before creating the Source, identify the Hosted Collector you want to use or create a new Hosted Collector. For instructions, see [Configure a Hosted Collector](/docs/send-data/hosted-collectors/configure-hosted-collector).
+
+To configure a Druva Source:
+
+1. In Sumo Logic, select **Manage Data** > **Collection** > **Collection**. 
+1. On the Collectors page, click **Add Source** next to a Hosted Collector.
+1. Select **Druva** icon.  <br/>  <img src={useBaseUrl('img/send-data/druva-icon.png')} alt="druva-icon.png" width="120" />
+1. Enter a **Name** to display for the Source in the Sumo Logic web application. The description is optional. <br/>   <img src={useBaseUrl('img/send-data/druva-config.png')} alt="druva-config.png" width="400" />
+1. (Optional) For **Source Category**, enter any string to tag the output collected from the Source. Category metadata is stored in a searchable field called `_sourceCategory`.
+1. (Optional) **Fields**. Click the **+Add** to define the fields you want to associate. Each field needs a name (key) and value.
+   * ![green check circle.png](/img/reuse/green-check-circle.png) A green circle with a check mark is shown when the field exists in the Fields table schema.
+   * ![orange exclamation point.png](/img/reuse/orange-exclamation-point.png) An orange triangle with an exclamation point is shown when the field doesn't exist in the Fields table schema. In this case, an option to automatically add the nonexistent fields to the Fields table schema is provided. If a field is sent to Sumo Logic that does not exist in the Fields schema it is ignored, known as dropped.
+1. **Base URL**. Choose the base url from the drop-down menu as per your requirements — [Data Governance Cloud](https://apis.druva.com) and [Data GovernanceGov Cloud](https://govcloudapis.druva.com).<br/><img src={useBaseUrl('/img/send-data/druva-base-url.png')} alt="druva-base-url.png" width="400"/>
+1. **Client ID**. Enter your Client ID. To get Client ID, follow the instructions from [Create and Manage Druva API Credentials](https://docs.druva.com/Druva_Cloud_Platform/Integration_with_Druva_APIs/Create_and_Manage_API_Credentials).
+1. **Secret Key**. Enter your Secret Key. To get Secret Key, follow the instructions from [Create and Manage Druva API Credentials](https://docs.druva.com/Druva_Cloud_Platform/Integration_with_Druva_APIs/Create_and_Manage_API_Credentials).
+1. When you are finished configuring the Source, click **Save**.
+
+## Error types
```

Comment:
```suggestion
## Error Types
```

Reply:
Stet - [headers must be sentence case](https://help.sumologic.com/docs/contributing/style-guide/#capitalization)